modules/ROOT/pages/logging-in-mule.adoc

```diff
@@ -0,0 +1,331 @@
+= Configuring Logging
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: mule, studio, logger, logs, log, notifications, errors, debug
+
+You can configure what gets logged, where it gets logged, and how by editing a configuration file that sits in your project.
+
+Mule logs multiple messages and specific elements in your applications' flows to help you debug and keep track of events. You can also include the xref:logger-component-reference.adoc[`<logger>` Element] anywhere in a flow and set it up to output any message you want. By creating a configuration file, you can define what kinds of messages to log, in what way (asynchronously or synchronously), and where they get logged (such as to the console, to disk, to an endpoint or to a database).
+
+For logging, Mule uses http://www.slf4j.org/[slf4j], which is a logging facade that discovers and uses a logging strategy from the classpath, such as https://logging.apache.org/log4j/2.x/[log4j2] or the JDK Logger. By default, Mule includes log4j2, which is configured with a file called `log4j2.xml`.
+
+The Mule server has a `log4j2.xml` in its `conf` directory, which you can customize when running the server in standalone mode. This config file is is relevant for data output to the `mule_ee.log` file.
+
+
+If you choose to remove the log4j2.xml file from your application's classpath, Mule Runtime applies a default logging configuration for the application, and outputs the log to MULE_HOME/logs/<application-name>.log.
```

Comment:
backticks around file names, for example:
```
`log4j2.xml`
```

Reply:
maybe "application, and " -> "application and "

backticks needed around `MULE_HOME/logs/<application-name>.log`